### PR TITLE
Story5 integrate timer

### PR DESCRIPTION
**Pull request by ellie-r**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,91 @@
...
+        <div class="line horizontal lineTwo thick"></div>
+        <div class="line vertical lineThree thick"></div>
+        <div class="line horizontal lineFour thick"></div>
+        <div class="line vertical lineFive thick"></div>
+        <div class="line horizontal lineSix thin"></div>
+        <div class="line vertical lineSeven thin"></div>
+        <div class="line horizontal lineEight thick grimReaperZone"></div>
+        <div class="line horizontal lineNine thick"></div>
+        <div class="line vertical lineTen thick"></div>
+        <div class="line horizontal lineEleven"></div>
```
Comment (marcusxavier85) [inline index.html]: why?

File: index.html [hunk 2]
```diff
@@ -10,12 +10,91 @@
...
+        </div>
+        <div class="tombstone">
+            <img src="images/tombstone.png" alt="tombstone">
+        </div>
+        <div id="timer">
+            <p>00:45</p>
+        </div>
+    </div>
+  </div>
+
```
Comment (marcusxavier85) [inline index.html]: why?

File: script.js [hunk 3]
```diff
@@ -1,13 +1,143 @@
...
+    reaperMove.cancel();
+    grimReaperZone.removeEventListener('mouseleave', moveReaper);
+
+}
+
+// code here to activate win modal displaying
+function win() {
+    winModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
```
Comment (marcusxavier85) [inline script.js:74]: why can't line 73 and 74 be in the funciton reset game?

File: script.js [hunk 4]
```diff
@@ -1,13 +1,143 @@
...
+    resetGame();
+}
+
+// code here to activate lose modal displaying
+function lose() {
+    loseModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
```
Comment (marcusxavier85) [inline script.js:82]: same as above

File: script.js [hunk 5]
```diff
@@ -1,13 +1,143 @@
...
-/* timer for game code */
 let timerRunning = false;
-if ( timerRunning === false) {
-    startTimer(5, document.querySelector('#timer p'));
+
+//call main function to start game from here
+function runGame() {
+    ghostMove.play();
+    pumpkinMove.play();
+    if ( timerRunning === false) {
```
Comment (marcusxavier85) [inline script.js]: space in between bracket and timerRunning

File: script.js [hunk 6]
```diff
@@ -1,13 +1,143 @@
...
+        startTimer(45, document.querySelector('#timer p'));
+    }
+    listenForCollisions();
+    listenForWinning();
+    grimReaperZone.addEventListener('mouseleave',moveReaper);
 }
+
+let timer;
```
Comment (marcusxavier85) [inline script.js:136]: why?

File: script.js [hunk 7]
```diff
@@ -1,13 +1,143 @@
...
 function startTimer(duration, element) {
     timerRunning = true;
     let timeLeft = duration;
-    let timer = setInterval( () => {
+    timer = setInterval( () => {
```
Comment (marcusxavier85) [inline script.js]: space between bracket and anonymous func is not consistent with rest of code

File: style.css [hunk 8]
```diff
@@ -1,4 +1,352 @@
...
+}
+
+.endButton {
+    width: 100px;
+    padding: 10px;
+    font-size: 2rem;
+    cursor: pointer;
+    font-family: "Creepster";
+}
+
```
Comment (marcusxavier85) [inline style.css:348]: #timer p margin: 0; why?

Comment (ellie-r) [reply style.css:348]: as p has a default margin which created space between the game box and title

Comment (marcusxavier85) [reply style.css:348]: chill one thats safe

**Pull request merged**